auto/cc/test

```diff
@@ -79,7 +79,7 @@ case $NXT_CC_NAME in
 
         # -O2 enables -fstrict-aliasing and -fstrict-overflow.
         #NXT_CFLAGS="$NXT_CFLAGS -O2"
-        #NXT_CFLAGS="$NXT_CFLAGS -Wno-strict-aliasing"
+        NXT_CFLAGS="$NXT_CFLAGS -fno-strict-aliasing"
```

Comment:
I would be happier if the commit message explicitly says that strict aliasing is not welcome, and that this is only a precaution due to hysterical raisins.  :)

Reply:
The thing is, I _do_ like being able to alias memory, I explicitly chose to do it in the wasm module (and disabled strict-aliasing there).

So while I won't encourage it, I am also reluctant to blatantly discourage it...

Reply:
The standard still allows you to alias memory.  It just says you should do it via unions or memcpy.  Hmm, yeah, those might not be as convenient as casts some times.

Maybe `-fno-strict-aliasing` has its place in some code, I admit.  I'm not a fan of it, but hey, it's simple.

Reply:
I guess if some small doses of no-strict-aliasing are good, and then we don't want to blatantly discourage it.  Let's not discourage it in the commit message, and just be careful with it.